api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -3991,6 +3992,50 @@ message ElasticsearchRequest {
   string Query = 12 [(gogoproto.jsontag) = "query"];
 }
 
+// OpenSearchRequest is emitted when a user executes a OpenSearch request via database-access.
+message OpenSearchRequest {
```

Comment:
Could you provide aligment for a new message is webapps  repo to cover the new message ?

Reply:
The PR for this is coming today.